components/clientComponents/globals/Header/Header.tsx

```diff
@@ -45,34 +41,18 @@ export const Header = ({ context = "default", className }: HeaderParams) => {
               </div>
             </a>
           </Link>
-
-          {isDefault && (
-            <div className="mt-3 box-border block h-[40px] px-2 py-1 text-xl font-semibold">
-              {t("adminNav.allForms", { ns: "common" })}
-            </div>
-          )}
-          {isAdmin && (
-            <div className="mt-3 box-border block h-[40px] px-2 py-1 text-xl font-semibold">
-              {t("adminNav.allForms", { ns: "admin-login" })}
-            </div>
-          )}
-          {isFormBuilder && status === "authenticated" && (
-            <div className="pt-3">
-              <Link href={`/${language}/forms`}>{t("adminNav.allForms", { ns: "common" })}</Link>
-              <span className="mx-2 inline-block"> / </span>
-            </div>
```

Comment:
I made the assumption we only want to show the Forms link if a user is authenticated and only show the input on the form-builder pages